main/convert.js

```diff
@@ -185,6 +188,39 @@ const convertToWebm = PCancelable.fn(async (options, onCancel) => {
   ]);
 });
 
+const convertToAv1 = PCancelable.fn(async (options, onCancel) => {
+  if (options.isMuted) {
+    const muteProcess = mute(options.inputPath);
+
+    onCancel(() => {
+      muteProcess.cancel();
+    });
+
+    options.inputPath = await muteProcess;
+  }
+
+  return convert(options.outputPath, options, [
+    '-i', options.inputPath,
+    '-r', options.fps,
+    ...(
+      options.shouldCrop || !areDimensionsEvent(options) ? [
+        '-s', `${makeEven(options.width)}x${makeEven(options.height)}`,
+        '-ss', options.startTime,
+        '-to', options.endTime
+      ] : []
+    ),
+    '-c:v', 'libaom-av1',
+    '-c:a', 'libopus',
+    '-crf', '34',
+    '-b:v', '0',
+    '-strict', 'experimental',
+    '-cpu-used', '4',
+    '-row-mt', '1',
+    '-tiles', '2x2',
```

Comment:
These are not necessary but they do help a bit with how long it takes for me. If for some reason they don't work, we can remove later.

Reply:
Can you add a short code comment about what it does?